app/src/main/cpp/src/components/transport_manager/include/transport_manager/bluetooth_le/bluetooth_le_device.h

```diff
@@ -44,70 +44,31 @@
 namespace transport_manager {
 namespace transport_adapter {
 
-/* BD Address */
-typedef struct {
-	uint8_t b[6];
-} __attribute__((packed)) bdaddr_t;
-
 /**
  * @brief Information about device that use bluetooth transport.
  */
 class BluetoothLeDevice : public Device {
  public:
-  /**
-   * @brief Return device unique identifier.
-   *
-   * @return string with device unique identifier.
-   */
-  static std::string GetUniqueDeviceId(const bdaddr_t& device_address);
 
   /**
    * @brief Constructor.
    *
-   * @param address Bluetooth address.
-   * @param name Human-readable device name.
-   * @param rfcomm_channels List of RFCOMM channels where SmartDeviceLink
-   *service has been discovered.
+   * @param device_address Bluetooth address.
+   * @param device_name Human-readable device name.
    **/
-  BluetoothLeDevice(const bdaddr_t& device_address,
+  BluetoothLeDevice(const std::string& device_address,
                   const char* device_name);
 
-  /**
-   * @brief Compare devices.
-   *
-   * This method checks whether two SBluetoothDevice structures
-   * refer to the same device.
-   *
-   * @param other Device to compare with.
-   *
-   * @return true if devices are equal, false otherwise.
-   **/
   bool IsSameAs(const Device* other) const override;
 
-  bool GetRfcommChannel(const ApplicationHandle app_handle,
-                        uint8_t* channel_out);
-
-  /**
-   * @brief Update list of applications available on device.
-   *
-   * @return Container with list of applications.
-   */
   ApplicationList GetApplicationList() const override;
 
-  /**
-   * @brief Return device bluetooth address.
-   *
-   * @return Device bluetooth address.
-   */
-  const bdaddr_t& address() const {
-    return address_;
-  }
-
  private:
   /**
    * @brief Device bluetooth address.
    **/
-  bdaddr_t address_;
+  std::string address_;
```

Comment:
@VadymLuchko I think you can keep `bdaddr_t` type and use functions from <bluetooth.h> like `str2ba` and `ba2str` for making a conversion

Reply:
Seems <bluetooth.h> is a part of bluez which we do not use in android build